joomla/joomla-cms
# [4.0] Tidy up spacing on plugin descriptions

Pull Request for Issue #31267 

### Summary of Changes
- ~~Changes `<br>` to `<br />` consistently throughout plugin language files.~~ 
- ~~Changes `<br>` to `<br><br>` in various language files to add an additional line before warnings in descriptions to improve readability on screen.~~
- Changes `<br>` to paragraphs in various language files to add spacing and formatting before warnings in descriptions to improve readability on screen.

Where @gostn identified Content - Fields plugin having the issue in #31267 a closer inspection of plugins showed a number of plugins with descriptions that could be improved by additional spacing being added.

### Testing Instructions
- Before applying the patch, check a few of the plugins in the list to see what the before status looks like.
- Apply the patch
- Check the various plugins listed in the file changes for this PR to see the various changes have taken effect without impacting the usability of the plugin details.

### Actual result BEFORE applying this Pull Request
Before spacing is improved, there will not be a space between the first paragraph and the second paragraph:
![image](https://user-images.githubusercontent.com/5515866/97778205-c4db1280-1bc9-11eb-875b-3c85821f1310.png)

### Expected result AFTER applying this Pull Request
After spacing is improved, there will be an additional space between the first paragraph and the second paragraph:
![image](https://user-images.githubusercontent.com/5515866/97778216-d6241f00-1bc9-11eb-875a-2387efb4cbd0.png)

### Documentation Changes Required
None
